test/dynamo/test_dynamo.py

```diff
@@ -95,6 +95,10 @@ def train_model(self, model, data, target):
   def run_model_with_dynamo(self, model, data, target):
     return self.train_model(model, data, target)
 
+  @unittest.skipIf(
+      os.environ.get('PYTORCH_XLA_TESTS_SKIP'),
```

Comment:
NB: Can we rename this to something more specific? For example `PTXLA_SKIP_TESTING_DEV_FEATURES`? Right now, the name makes it sounds like we skip all tests.

Reply:
Thanks, Steven! will do

Reply:
updated in https://github.com/pytorch/xla/pull/4533 and will track in https://github.com/pytorch/xla/pull/4533, this PR will be closed due to duplicate